docs/content/misc/faq.ngdoc

```diff
@@ -130,8 +140,8 @@ Yes. See instructions in {@link downloading}.
 ### What browsers does AngularJS work with?
 
 We run our extensive test suite against the following browsers: the latest versions of Chrome,
-Firefox, Safari, and Safari for iOs, as well as Internet Explorer versions 9-11. See {@link guide/ie
-Internet Explorer Compatibility} for more details on supporting legacy IE browsers.
+Firefox, Safari, and Safari for iOs, Edge, as well as Internet Explorer versions 9-11. See
```

Comment:
I removed it again. Edge had some problems with the script security checks last time i checked